packages/query/src/bitcoin/transaction/use-check-utxos.ts

```diff
@@ -34,124 +19,3 @@ export function filterOutIntentionalUtxoSpend({
     });
   });
 }
-
-interface CheckInscribedUtxosByBestinslotArgs {
-  inputs: TransactionInput[];
-  txids: string[];
-  client: BitcoinClient;
-}
-
-async function checkInscribedUtxosByBestinslot({
-  inputs,
-  txids,
-  client,
-}: CheckInscribedUtxosByBestinslotArgs): Promise<boolean> {
-  /**
-   * Get the list of inscriptions moving on a transaction
-   * @see https://docs.bestinslot.xyz/reference/api-reference/ordinals-and-brc-20-and-bitmap-v3-api-mainnet+testnet/inscriptions
-   */
-  const inscriptionIdsList = await Promise.all(
-    txids.map(id => client.BestInSlotApi.getInscriptionsByTransactionId(id))
-  );
-
-  const inscriptionIds = inscriptionIdsList.flatMap(inscription =>
-    inscription.data.map(data => data.inscription_id)
-  );
-
-  const inscriptionsList = await Promise.all(
-    inscriptionIds.map(id => client.BestInSlotApi.getInscriptionById(id))
-  );
-
-  const hasInscribedUtxos = inscriptionsList.some(resp => {
-    return inputs.some(input => {
-      if (!input.txid) throw new Error('Transaction ID is missing in the input');
-      const idWithIndex = `${bytesToHex(input.txid)}:${input.index}`;
-      return resp.data.satpoint.includes(idWithIndex);
-    });
-  });
-
-  return hasInscribedUtxos;
-}
-
-export function useCheckUnspendableUtxos(blockTxAction?: () => void) {
```

Comment:
How can this be relocated, but `useBitcoinBroadcastTransaction`? Isn't it used there?